features/0023-did-exchange/README.md

```diff
@@ -482,7 +346,7 @@ When an out of band invitation is received containing a public DID for which the
   "@type": "https://didcomm.org/didexchange/1.0/reuse",
   "@id": "12345678900987654321",
   "~thread": {
-    "pthid": "<The Parent Thread ID of the Out Of Band Invitation>"
+    "pthid": "<The Parent Thread ID of the Out-of-Band invitation>"
```

Comment:
Same change as above -- remove Parent Thread.

Reply:
This shouldn't be removed, I think.

Reply:
The data value to go there is the "ID of the OOB Invitation" not the "Parent Thread ID of the OOB Invitation".  There is no Parent Thread ID in the invitation.

Reply:
Oh, whoops. You're right. `pthid` should contain the `@id` property of the OOB invitation, because as you say, there's nothing else to use. Thanks for catching me.